wagtail_localize_rws_languagecloud/templatetags/wagtaillocalizerwslanguagecloud_tags.py

```diff
@@ -0,0 +1,38 @@
+from django import template
+
+from ..models import LanguageCloudFile
+
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def language_cloud_statuses(context):
```

Comment:
This function name is not super descriptive. My first thought was this would just give us the list of all possible statuses, but also it is a tricky function to come up with a clear concise name for. Naming things is hard :)

How about `language_cloud_statuses_for_page()` ?

Reply:
That makes sense. I'll change this.